<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>453</width>
        <height>513</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>본문</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>태그</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>수정</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>가격</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>..</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>제목</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>대표이미지</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>메뉴명</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>카페</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
